src/layer-composer/generators/background/background.ts

```diff
@@ -1,16 +1,13 @@
+import { BackgroundPaint, BackgroundLayout } from 'mapbox-gl'
+import { ExtendedLayer, Group } from '../../../types/types'
```

Comment:
There is support for absolute imports so I think worth it to use it and avoid hard to read imports
```suggestion
import { ExtendedLayer, Group } from 'types/types'
```

Also maybe a more specific name for the file helps as `types` sounds too generic

Reply:
I agree but that doesn't work :(
I have changed `types/types` to just `types` though